src/connectors/toggleRefinement/connectToggleRefinement.js

```diff
@@ -37,8 +37,8 @@ Full documentation available at https://community.algolia.com/instantsearch.js/v
 
 /**
  * @typedef {Object} CustomToggleWidgetOptions
- * @property {string} attributeName Name of the attribute for faceting (eg. "free_shipping").
- * @property {string} label Human-readable name of the filter (eg. "Free Shipping").
+ * @property {string} attribute Name of the attribute for faceting (eg. "free_shipping").
+ * @property {string} [label] Human-readable name of the filter (eg. "Free Shipping").
```

Comment:
`label` doesn't exist anymore.

Reply:
I got confused. I will update the specs as it's missing.

Reply:
Actually no, `label` is replaced by `templates.labelText`.

See https://github.com/algolia/instantsearch.js/pull/3135/files#diff-e4aa96a4cb05bd6587f30e4b2ce5f9dfR407.

WDYT?

Reply:
I'm fine with it as soon we have a way to customise the label. Since we have the templates in InstantSearch it makes more sense to put it there rather than a top level options. Why do we go for `templates.labelText` vs `templates.label` ?

Reply:
Yes – would you go for `templates.label` instead? Happy to go both ways.

Reply:
I'm fine with the consistency!